woocommerce/woocommerce-ios
# Store creation M2: success screen

Part of #8118


### Description


This PR included just a new screen, `StoreCreationSuccessView` and its hosting controller, which is shown once the site becomes Atomic after a successful purchase of an eCommerce plan. I was hoping to integrate with IAP before this PR, but there is a backend issue and I haven't been able to make a successful IAP yet. In the next PR, I'm going to make a mock IAP implementation so that we can test the whole flow more easily.

The preview view is using `WebView` to show the given site URL, with interactions disabled. The preview CTA brings up a Safari sheet to see the site in fullscreen.

### Testing instructions


Please feel free to check the SwiftUI previews, or modify the code to see this screen in action wherever there's a `Site` available. For example, you can replace the [code in the store picker](https://github.com/woocommerce/woocommerce-ios/blob/084d5acb254ffa4737762b8700433fc155af96e0/WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift#L800-L822) with:

```swift
        guard let url = URL(string: site.url) else {
            return
        }
        let successView = StoreCreationSuccessHostingController(siteURL: url) {}
        navigationController?.pushViewController(successView, animated: true)
```

### Screenshots


light | dark | preview
-- | -- | --
![Simulator Screen Shot - iPhone 13 Pro - 2022-11-17 at 14 27 48](https://user-images.githubusercontent.com/1945542/202374063-e4376555-f702-4f15-a3f6-2e0357b19fc8.png) | ![Simulator Screen Shot - iPhone 13 Pro - 2022-11-17 at 14 28 07](https://user-images.githubusercontent.com/1945542/202374077-8bf61d1c-a329-4f3f-a2f0-1abf7ecd808c.png) | ![Simulator Screen Shot - iPhone 13 Pro - 2022-11-17 at 14 28 20](https://user-images.githubusercontent.com/1945542/202374087-79912724-4eb1-4805-87be-36601122032a.png)


---
